0000-catalog.md

```diff
@@ -710,7 +716,8 @@ message CatalogProductSetStatusAction {
 
 Validation requirements:
 
-- If a catalog_product with catalog_product_id does not exist the transaction is
+- The catalog_product must be able to be updated or the transaction is invalid
+- If a catalog_product with product_id does not exist the transaction is
```

Comment:
This should clarify that if ANY of the catalog_products referenced in the transaction do not exist the transaction is invalid.

Reply:
✅